.travis.yml

```diff
@@ -7,7 +7,11 @@ php:
     - 7.1
 
 before_script:
-  - travis_retry composer update --no-interaction
+    - travis_retry composer update --no-interaction
 
 script:
-    ./vendor/bin/phpunit --testsuite tests
+    - vendor/bin/phpcs -p --warning-severity=0 src/ tests/
+    - vendor/bin/phpunit --coverage-clover=./tests/report/coverage.clover
+
+after_script:
+    - ./build/coverage_to_scrutinizer.sh
```

Comment:
I have changed this to:

```yaml
after_script:
  - test -f ./tests/report/coverage.clover && (wget https://scrutinizer-ci.com/ocular.phar; php ocular.phar code-coverage:upload --format=php-clover ./tests/report/coverage.clover)
```

https://github.com/graze/dog-statsd/blob/master/.travis.yml#L18